### DISCUSSION TITLE
Device Friendly Name

### DISCUSSION THREAD

**Discussion by MiguelGillesIT** [chmorgan/sharppcap#418] [Q&A] [answered]
Hello, I'm working on a network analyzer application based on a GUI. Use name property to choose a device isn't very common. I wanna know if there is another property who can help me on this way.

**Comment by kayoub5** [comment, marked as answer]
It is called `device.Interface.FriendlyName`

https://github.com/dotpcap/sharppcap/blob/d7a5e50497f0ee5a4e4d5f4b01350473b8b3d9ab/SharpPcap/LibPcap/PcapInterface.cs#L46-L49

**Reply by MiguelGillesIT** [reply]
Is it possible to access it by using ICaptureDevice object ?

**Reply by kayoub5** [reply]
no, not all devices provide a pcap interface (since it could be a windivert device for example)

**Reply by MiguelGillesIT** [reply]
Thanks
